### PR TITLE
Translate Pokemon and Moves + German Translation

### PR DESCRIPTION
**Pull request by 123FLO321**
This PR adds the ability to translate Pokemon and Movesets and also adds German translations.

I am currently using this and it seems to be working just fine but still needs more testing.

### CODE REVIEW

**Comment by Falaris01** [comment]
Thank you for this :)

Just tried it out, I noticed there are a few translations missing:

image

image

image

Have not tested it thorough yet

File: src/main/resources/Messages_de.properties [hunk 1]
```diff
@@ -0,0 +1,1130 @@
...
+PauseMessage=Ihre Benachrichtigungen wurden GESTOPPT. Sie erhalten keine Benachrichtigungen, bis Sie den Befehl `<unaupusecommand>` verwenden.
+UnPauseCommand=!unpause
+UnPauseMessage=Ihre Benachrichtigungen sind derzeit AKTIV. Sie erhalten Benachrichtigungen, bis Sie den Befehl `<pausecommand>` verwenden.
+SettingsCommand=!settings
+JoinRaidCommand=!joinraid
+JoinRaidLobbyMessage=Du wurdest in <channel> platziert. Es gibt jetzt <lobbysize> Benutzer in der Lobby.
+NoRaidLobbyMessage=Entschuldigung, es gibt keine aktiven Raid Lobbies mit dem Lobby Code `<lobbycode>`.
+AlreadyInLobbyMessage=Du bist bereits in dieser Raid-Lobby!
+AlertRaidChatsMessage=<user> trat <boss-or-egg> in <channel> bei. Es gibt jetzt <membercount> Benutzer in der Lobby. Treten Sie der Lobby bei, indem Sie auf das ? klicken oder indem Sie `<jouchraidcommand> <lobbycode>` eingeben.
+RaidEndSoonMessage=Der Raid bald enden. Verbleibend:
```
Comment (Falaris01) [inline src/main/resources/Messages_de.properties]: Der Raid endet bald. Verbleibend:

File: src/main/resources/Messages_de.properties [hunk 2]
```diff
@@ -0,0 +1,1130 @@
...
+RaidHasEndedMessage=Der Raid ist vorbei. Die Lobby wird bald geschlossen. Verbleibend:
+RaidLobbyHelpStart=Meine Raid-Lobby-Befehle sind:
+EggHatchedMessage=Das Raid-Ein ist geschl�pft. Es ist ein
```
Comment (Falaris01) [inline src/main/resources/Messages_de.properties]: Raid-Ei

File: src/main/resources/Messages_de.properties [hunk 3]
```diff
@@ -0,0 +1,1130 @@
...
+ResetRaidsMessage=Alle Raid-Benachrichtigungseinstellungen wurden zur�ckgesetzt.
+ResetPresetsCommand=!resetpresets
+ResetPresetsMessage=Alle Benachrichtigungs-Voreinstellungen wurden zur�ckgesetzt.
+SuburbsCommand=!suburbs
+SuburbListCommand=!suburblist
+SuburbListMessageStart=M�gliche Vororte sind:
+RegionsCommand=!regions
+RegionListCommand=!regionlist
+RegionListMessageStart=M�gliche Regionen sind:
+NotPossibleRaidBossesError=Das folgende Pokemon, das du eingegeben hast, sind keine Raidbosse:\n\n
```
Comment (Falaris01) [inline src/main/resources/Messages_de.properties]: "Die folgenden Pokemon, die du..." oder "...hast, ist kein Raidboss:"

File: src/main/resources/Messages_de.properties [hunk 4]
```diff
@@ -0,0 +1,1130 @@
...
+YouWillNowBeNotifiedOf=Sie werden jetzt �ber folgendes Benachrichtigt:
```
Comment (Falaris01) [inline src/main/resources/Messages_de.properties]: "über Folgendes benachrichtigt:"

**Comment by darkelement1987** [comment]
how do i start the novabot instance to use a specific translation? Do i use a parameter of as specification in the config file?

**Comment by 123FLO321** [comment]
@Falaris01 Thank you. I fixed the translation problems.
Also not everything is localizable yet. Everything that can be translated is translated in German now.

**Comment by 123FLO321** [comment]
@darkelement1987 add the`-l` flag to your launch options (e.g.: `-l de`)

**Pull request merged**